src/Utils/PricingUtils.ts

```diff
@@ -7,10 +7,11 @@ import { AutopilotTier } from "../Contracts/DataModels";
  * Otherwise, return numberOfRegions
  * @param number
  */
-export function normalizeNumber(number: any): number {
-  const normalizedNumber: number = number === null ? 0 : isNaN(number) ? 0 : parseInt(number);
-
-  return normalizedNumber;
+export function normalizeNumber(number: null | undefined | string | number): number {
+  if (!number) {
+    return 0;
+  }
+  return Math.floor(Number(number));
```

Comment:
Judging by the unit tests, the floor is expected. Let me know if you think otherwise.